doc/combining.rst

```diff
@@ -13,6 +13,8 @@ Combining data
 
 * For combining datasets or data arrays along a dimension, see concatenate_.
 * For combining datasets with different variables, see merge_.
+# For combining datasets with different coordinates or missing values,
+  or data arrays with outer-join alignment, see combine_.
```

Comment:
I would drop "or data arrays with outer-join alignment" -- it's a little confusing.